cloudbridge/cloud/providers/gce/services.py

```diff
@@ -394,10 +397,133 @@ def list(self, limit=None, marker=None):
                                      limit=limit, marker=marker)
 
 
+class GCEInstanceService(BaseInstanceService):
+
+    def __init__(self, provider):
+        super(GCEInstanceService, self).__init__(provider)
+
+    def create(self, name, image, instance_type, zone=None,
```

Comment:
There is a "network" parameter in the create method of the InstanceService interface, after instance_type, before zone.